setup_scripts/setup_udev_rules.sh

```diff
@@ -29,3 +29,25 @@ echo "================================================================"
 echo "Finished Installing snowbots udev rules"
 echo "================================================================"
 
+echo "================================================================"
+echo "Adding phidgets udev rules to udev folder..."
+echo "================================================================"
+
+
+# Target file directory
+FILE="/etc/udev/rules.d/99-phidgets.rules"
+if [ -e "$FILE" ]
+then
+  echo "Delete the existing Snowbots udev rules"
```

Comment:
should be `Deleting the existing Phidgets udev rules`?

Reply:
Yep, cleaned up.